en/get-started.md

```diff
@@ -0,0 +1,735 @@
+    *************************** 1. row ***************************
+              TYPE: tidb
+          INSTANCE: basic-tidb-1.basic-tidb-peer.tidb-cluster.svc:4000
+    STATUS_ADDRESS: basic-tidb-1.basic-tidb-peer.tidb-cluster.svc:10080
+           VERSION: 5.7.25-TiDB-v4.0.0
+          GIT_HASH: 689a6b6439ae7835947fcaccf329a3fc303986cb
+        START_TIME: 2020-05-28T22:49:17Z
+            UPTIME: 4m15.459086883s
+    *************************** 2. row ***************************
+              TYPE: tidb
+          INSTANCE: basic-tidb-0.basic-tidb-peer.tidb-cluster.svc:4000
+    STATUS_ADDRESS: basic-tidb-0.basic-tidb-peer.tidb-cluster.svc:10080
+           VERSION: 5.7.25-TiDB-v4.0.0
+          GIT_HASH: 689a6b6439ae7835947fcaccf329a3fc303986cb
+        START_TIME: 2020-05-28T22:50:11Z
+            UPTIME: 3m21.459090928s
+    *************************** 3. row ***************************
+              TYPE: pd
+          INSTANCE: basic-pd:2379
+    STATUS_ADDRESS: basic-pd:2379
+           VERSION: 4.0.0
+          GIT_HASH: 56d4c3d2237f5bf6fb11a794731ed1d95c8020c2
+        START_TIME: 2020-05-28T22:45:04Z
+            UPTIME: 8m28.459091915s
+    *************************** 4. row ***************************
+              TYPE: tikv
+          INSTANCE: basic-tikv-1.basic-tikv-peer.tidb-cluster.svc:20160
+    STATUS_ADDRESS: 0.0.0.0:20180
+           VERSION: 4.0.0
+          GIT_HASH: 198a2cea01734ce8f46d55a29708f123f9133944
+        START_TIME: 2020-05-28T22:47:41Z
+            UPTIME: 5m51.45910075s
+    *************************** 5. row ***************************
+              TYPE: tikv
+          INSTANCE: basic-tikv-0.basic-tikv-peer.tidb-cluster.svc:20160
+    STATUS_ADDRESS: 0.0.0.0:20180
+           VERSION: 4.0.0
+          GIT_HASH: 198a2cea01734ce8f46d55a29708f123f9133944
+        START_TIME: 2020-05-28T22:48:21Z
+            UPTIME: 5m11.459102648s
+    *************************** 6. row ***************************
+              TYPE: tikv
+          INSTANCE: basic-tikv-2.basic-tikv-peer.tidb-cluster.svc:20160
+    STATUS_ADDRESS: 0.0.0.0:20180
+           VERSION: 4.0.0
+          GIT_HASH: 198a2cea01734ce8f46d55a29708f123f9133944
+        START_TIME: 2020-05-28T22:47:17Z
+            UPTIME: 6m15.459103932s
+    6 rows in set (0.01 sec)
+    ```
+
+4. Load Grafana dashboard
+
+    As done above for port 4000 to the TiDB service, also forward the port for Grafana so we can load the monitoring dashboard:
+
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    kubectl port-forward -n tidb-cluster svc/basic-grafana 3000 > pf3000.out &
+    ```
+
+    The dashboard will be accessible at <http://localhost:3000> on the host where you've run `kubectl`. Note that if you're running `kubectl` in a Docker container or on a remote host, you may not be able to load this URL and additional networking beyond the scope of this document may be necessary.
+
+    The default username and password in Grafana are both "admin".
```

Comment:
We need to clear the `port-forward` in steps 2 & 4.

Reply:
What do you mean by "clear"?

Reply:
The port-forward process is started in the background, after we're done with the demo access, we'd better stop the port-forward process, otherwise, it will always run and when you use this command to port-forward again or for another cluster, it will fail.

Reply:
Oh, OK. In my experience, it often exits on its own, but this is good advice.